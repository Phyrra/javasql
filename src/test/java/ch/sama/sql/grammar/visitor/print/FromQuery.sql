select *
from (
    select 1
) as t