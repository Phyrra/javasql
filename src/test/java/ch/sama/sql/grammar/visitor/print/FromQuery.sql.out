statement
	select
		value
			all
	from
		source
			aliased
				statement
					select
						value
							primary
								numeric: 1
				t
